docs/install.md

```diff
@@ -6,7 +6,33 @@
 
 1. Grant `cluster-admin` permissions to the current user.
 
-## Installing from a local clone of this repository (only developer preview level support)
+## Installation (developer preview)
+
+### Installing from the release page
+
+1. Run the following command
+
```

Comment:
add a link to https://github.com/openshift/csi-driver-projected-resource/releases so then know what are their options for setting RELEASE_VERSION

Reply:
updated